packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -177,6 +197,16 @@ const getOrphanPinpointApplications = async (account: AWSAccountInfo, region: st
   return apps;
 };
 
+/**
+ * Get all AppSync Apis in the account, and filter down to the ones we consider stale.
+ */
+const getOrphanAppSyncApis = async (account: AWSAccountInfo, region: string): Promise<AppSyncApiInfo[]> => {
+  const appSyncClient = new aws.AppSync(getAWSConfig(account, region));
+  const listApisResponse = await appSyncClient.listGraphqlApis({ maxResults: 25 }).promise();
```

Comment:
tested with 25 in ssh session.
![image](https://user-images.githubusercontent.com/5849952/209739143-d620b845-5f38-4c4a-ae19-db1aa2d2ba82.png)